lib/ui/painting/codec.cc

```diff
@@ -69,18 +70,24 @@ fxl::RefPtr<Codec> InitCodec(sk_sp<SkData> buffer, size_t trace_id) {
   TRACE_EVENT0("blink", "InitCodec");
 
   if (buffer == nullptr || buffer->isEmpty()) {
+    FXL_LOG(ERROR) << "InitCodec failed - buffer was empty ";
     return nullptr;
   }
 
   std::unique_ptr<SkCodec> skCodec = SkCodec::MakeFromData(buffer);
   if (!skCodec) {
+    FXL_LOG(ERROR) << "SkCodec::MakeFromData failed";
     return nullptr;
   }
-  if (skCodec->getFrameCount() > 1) {
+  if (skCodec->getFrameCount() > 1
+      // Temporarily disable WebP animations due to:
+      // https://github.com/flutter/flutter/issues/13017
```

Comment:
Please add a `// TODO(amirh)` here.

Reply:
I completely removed this as the Skia fix landed